### ISSUE TITLE
Non-functional Chrome addon called as IE tab is not working on the Firefox 57  with Chrome store foxified version 3.3

### ISSUE THREAD

**Issue by sanket9918** [Noitidart/Chrome-Store-Foxified#93]
I definitely have faith on your analysis of all the errors and and I definitely think that you will be able to resolve the error as soon as possible because tomorrow there is the launch of the Firefox 57 and would like to see your extension in action

Kindly take a look on the errors that encountered during our process at this website 

https://addons.mozilla.org/en-US/developers/upload/902096a0aa414ce587dbae34800195e9

**Comment by sanket9918** [comment]
As I followed from one of your issue tracker that you have told to load the add on or the extension in temporary debugging mode so I didn't and I got the following error 
![capture](https://user-images.githubusercontent.com/33617196/32713652-3987c3ae-c870-11e7-9355-c5ee247197d3.JPG)

**Comment by PAVAN9497** [comment]
I'm facing this problem while adding extension in Firefox
![screenshot_2017-11-13-14-00-00-603_org mozilla firefox](https://user-images.githubusercontent.com/33501957/32716586-7d757f10-c84e-11e7-8a72-3c1c4e855dbd.png)

**Comment by PAVAN9497** [comment]
I'm facing this problem while adding extension in Firefox

**Comment by Noitidart** [comment]
Hi folks, I don't have control over the extensions. I only can control Chrome Store Foxified extension. If that extension doesn't work, it needs to be tweaked, either by author of that addon, or by the community. I created a forum, if you post about it, hopefully others with time will join in and share their tweaks. I wish I did but I don't have the resources to spend time on all the extensions that don't work. :(
